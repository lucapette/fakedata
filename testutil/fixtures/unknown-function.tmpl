{{Int 42 42}}  {{Madeup}}