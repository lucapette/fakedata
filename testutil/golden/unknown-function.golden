template: template:1: function "Madeup" not defined
